### PR TITLE
Update publishing guidelines

### LINKED ISSUE #1137: Update Editorial Guidelines for Managing Director Publishing
I think I'm the first one to push out a lesson since we decided that managing director would be coordinating pull requests for the publishing process. Accordingly, I've noticed some kinks in the editorial guidelines (holdovers from the old system) that would cause confusion if you're strictly following the text step by step. Also caught a couple errors in the metadata guidelines from when we condensed them. 

Changes to "Acceptance and Publication - Editorial Checklist"
- [x] Step 1 asks you to update the ph_authors.yml file. That's alright in theory, except in practice no one has been updating the ph-submissions version of that file for a long time (it's about half the length) of the one on the jekyll repo. So if we updated the ph-submissions file and copied it over we'd lose a lot of data. I'm going to clarify which one we should be editing to be the ph-submissions file, and I'll update the ph-submissions file to be up-to-date. Update - we'll just remove this file from ph-submissions and clarify this in the guidelines.  
- [x] Under step 3, the metadata guidelines. The example given for a slug isn't meaningful, as it needs to be a lower-cased and hyphenated path to the lesson. The old text was helpful and should be added - "**slug** should have the path to the lesson on the public PH site, which means the hyphenated text following programminghistorian.org/lessons/ (e.g. building-static-sites-with-jekyll-github-pages)"
- [x] Under step 3, the metadata guidelines. The date of the lesson should be updated to the date it was moved to the jekyll repo. That information dropped off, so there's no indication that the date should be changed during the publishing process. 
- [x] Step 4 wants you to find an image and upload it to the gallery folder. But this folder doesn't actually exist anywhere on the ph-submissions repo - it's a folder that exists purely on the jekyll repo. So if you are going to add it to the gallery folder you'd need to preemptively make a pull request before asking the ME to make a pull request, which doesn't make a lot of sense. So it sounds like we'll need to make a folder on the ph-submissions repo to fold new gallery photos that mirrors the structure of the Jekyll site. 
- [ ] Steps 5 and 6 should be switched. It doesn't make sense to incorporate it into the Twitter bot spreadsheet before publishing with the ME. You'll need a link to be able to do so, which should only be generated after publishing it. Also, there is always the off chance that the lesson could come up in the bot if there is a lag in the process before it's actually live. 

With those changes it should be a little more straightforward and clear about what is happening and what the managing editor's role will be. The lesson editor prepares everything on the ph-submissions repo, which is what I think we intended anyway. 
Do those changes sound alright @mdlincoln (particularly the ones that entail me making changes to the structure of ph-submissions)? If so I'll open a PR to change them.
@alsalin - when we're both back in the office after the break let's chat on slack when going through the final steps of the process to make sure we didn't miss anything.

### PR DESCRIPTION
**Pull request by walshbr**
Closes #1137.

@mdlincoln already approved the changes in ticket, so I don't think his review is required. 

Asking for a review from @alsalin since she will be the first person to actually follow the managing editor changes. Take a look and see if they make sense? I'll gather everything up for the FFMPEG lesson on the ph-submissions repo in just a couple minutes, and then you can take a swing at the publishing process whenever you're ready. Feel free to ping me by email or on slack if you wanted to talk while doing it. 

Once @alsalin approves the text here, the changes will need to be translated in Spanish (and potentially French, depending on where you're at in the process?), though the changes to the text itself are minimal. It's mostly moving things around.

### CODE REVIEW

**Review by alsalin — DISMISSED** [review]
It looks good in theory! Let's hope I can actually follow it. I may try to take a crack at the first lesson today.

**Comment by arojascastro** [comment]
All these changes concern new tasks for managing editors, so I guess the most suitable person in the Spanish team for translating this is @mariajoafana  - let us know if you need help.

**Comment by spapastamkou** [comment]
We can integrate the changes once we make a final review of the corresponding FR page before publishing the web site, you can go on with the rest @walshbr (if @mdlincoln sees no problem us doing this way)

**Comment by walshbr** [comment]
Oop I just noticed the tests were failing. Fixed, and they should be good for translating and then merging once Spanish is done!

**Comment by alsalin** [comment]
Questions/comments - now that I've gone through this twice (using the web GH), I realize the directions for dealing with file moving, copying, deleting are not clear. Mostly using "copy," we ask the ME to create new files in jekyll/, but should we add a line to delete the old files once all this moving, publishing, and archiving is done?

**Comment by walshbr** [comment]
@alsalin sorry just saw this. Which files were you talking about deleting? Was there something not captured below? 

```
Go to the directory for your local ph-submissions repository.
Add a new line to the YAML header of the now published lesson: redirect_from: "/lessons/LESSON-SLUG"
Copy the now published lesson from lessons/ into lessons/published/.
Copy the image folder containing the images for the now published lesson from images/ to images/published/.
Use git add, git commit, and git push to finalize all the changes (or follow the Making Technical Contributions instructions: https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions)

```
Also just a note to say that once we've wrapped the English portion of this I'll be stepping back for the sabbatical. Seems like @mariajoafana and the French team have plans for who will be translating this. Sounds like once Spanish translates this can be merged, and the French version will be incorporated once they go live?

**Comment by alsalin** [comment]
@walshbr the directions are for the command line and refer to "copying" rather than "moving" - in that sense, you end up with multiple copies in different folders

**Comment by walshbr** [comment]
Aaah I see what you're saying @alsalin. That does seem like a good catch. Would just changing the word "copy" to "move" do it?

**Comment by alsalin** [comment]
@walshbr i think so! as someone who is terrified of deleting things always, being clear on that language would help :P

**Comment by walshbr** [comment]
These changes should be good for English and can be merged once Spanish has them translated. I'll step back and sabbatical now. :)

**Comment by acrymble** [comment]
@mariajoafana you've been volunteered to do this translation. Are you able to do that or do you need someone else to take it on?

**Comment by mdlincoln** [comment]
I've just updated this rather old branch against the latest site, so it now has all the new French pages in it. I think this means that both @programminghistorian/spanish-team and @programminghistorian/french-team should now update their translations of the publishing guidelines to this branch?

File: en/editor-guidelines.md
Comment (mariajoafana) [inline en/editor-guidelines.md:203]: this section includes information on the lesson but no space for adding the actual bio

**Comment by mariajoafana** [comment]
I'm working on the translation into Spanish.

**Review by mariajoafana — DISMISSED** [review]

**Review by mdlincoln — APPROVED** [review]

**Pull request merged**
